tasks/actorstate/miner/miner.go

```diff
@@ -0,0 +1,143 @@
+package miner
+
+import (
+	"context"
+
+	"go.opentelemetry.io/otel"
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/lotus/chain/types"
+
+	miner "github.com/filecoin-project/lily/chain/actors/builtin/miner"
+	"github.com/filecoin-project/lily/tasks/actorstate"
+
+	"github.com/filecoin-project/lily/model"
+)
+
+// was services/processor/tasks/miner/miner.go
+
+// StorageMinerExtractor extracts miner actor state
+type StorageMinerExtractor struct{}
+
+func (m StorageMinerExtractor) Extract(ctx context.Context, a actorstate.ActorInfo, node actorstate.ActorStateAPI) (model.Persistable, error) {
```

Comment:
Is this used? I thought each extractor was run indepedently.

Reply:
Its not, I'll delete it